src/fmu/dataio/dataio.py

```diff
@@ -900,7 +900,7 @@ class InitializeCase:  # pylint: disable=too-few-public-methods
             or by other means.
         casename: Name of case (experiment)
         caseuser: Username provided
-        restart_from: ID of eventual restart
+        restart_from: ID of eventual restart (deprecated)
```

Comment:
If we remove it from the code it should not be in the docstring either. But suggest we do not remove it until having warned about it for a while. Purely theoretical that anyone is actually using it, but it is good practice to give some transition period when changing the API I think.

We can remove all the actual usage of it, but keep the input argument so that existing scripts will continue to work. Then just warn.